pom.xml

```diff
@@ -280,6 +282,66 @@
                     </excludes>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-checkstyle-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>com.puppycrawl.tools</groupId>
+                        <artifactId>checkstyle</artifactId>
+                        <version>8.41.1</version>
```

Comment:
(nitpick) Do you want to move the version number to the properties section?

Reply:
I don't see any reason why we will ever use it twice.

Reply:
Never mind... I somehow thought dependabot would only be able to bump version if it's defined in properties section.